src/ActivitiesSDKAdapter.js

```diff
@@ -24,46 +24,32 @@ import cache from './cache';
  */
 
 /**
- * Extracts JSON card and files from server activity.
+ * Maps SDK activity to adapter activity
  *
  * @private
  * @param {object} sdkActivity  SDK activity object
- * @returns {Array} Array of attachments.
+ * @returns {Activity} Adapter activity object
  */
-function parseSDKAttachments(sdkActivity) {
-  const {attachments = []} = sdkActivity;
-
-  try {
-    if (sdkActivity.object && sdkActivity.object.cards) {
-      sdkActivity.object.cards.forEach((c) => {
-        const card = JSON.parse(c);
+export function fromSDKActivity(sdkActivity) {
+  let card;
 
-        attachments.push({contentType: 'application/vnd.microsoft.card.adaptive', content: card});
-      });
+  if (sdkActivity.object && sdkActivity.object.cards) {
+    try {
+      card = JSON.parse(sdkActivity.object.cards[0]);
+    } catch (err) {
+      logger.warn('ACTIVITY', undefined, 'fromSDKActivity()', `Unable parse a card for activity with id "${sdkActivity.id}"`, err);
```

Comment:
you can put the activity id instead of undefined and shorten the error message: Unable to parse card